years_to_hours.rb

```diff
@@ -0,0 +1,20 @@
+def years_to_hours
```

Comment:
Consider splitting this up into separate functions.

Reply:
Yes, I see. Definitely makes more sense to break this up since its performing separate actions. Done.